schemas/iso19139/src/main/plugin/iso19139/update-fixed-info.xsl

```diff
@@ -469,17 +469,22 @@
       <xsl:variable name="ptLocale"
```

Comment:
Before creating the element line 466 (and at the textGroup level), we could check that the language is declared first. If a language is removed, all translations will also be removed. I could take care of this tomorrow if needed.